libs/net/dal/Migrations/1.0.161/Up/PostUp/00-ReportTemplate.sql

```diff
@@ -0,0 +1,334 @@
+                </td>
+
+                <td align="right">
+                  <span style="border: 1px solid #ccc; padding:4px 6px; text-transform: uppercase;">
+                    <span><a style="text-decoration:none; color: #6750a4; " href="#top">top</a></span>
+                    @if (hasPrev )
+                    {
+                    <span> | <a style="text-decoration:none; color: #6750a4; " href="#item-@prev">previous</a></span>
+                    }
+                    @if (hasNext )
+                    {
+                    <span> | <a style="text-decoration:none; color: #6750a4; " href="#item-@next">next</a></span>
+                    }
+                  </span>
+                </td>
+              </tr>
+            </table>
+
+            <h3 style="margin:16px 0px 10px 0px;">@(String.IsNullOrEmpty(sentiment) ? "" : $"{sentiment} ")@headline</h3>
+            <div style="padding: 2px 0px 8px 12px; width:100%; border-bottom:1px solid #E8E9F1; color:#584C50; font-size:.85rem; line-height:.9rem; text-transform:uppercase; margin-bottom:16px;">
+              <div style="display: block; font-size: 14px; line-height: 110%;">
+                <strong>@content.Source?.Name</strong><br/>
+                <strong>@content.PublishedOn?.AddHours(utcOffset).ToString("dddd, MMMM d yyyy")</strong><br/>
+                @if (!string.IsNullOrWhiteSpace(content.Page))
+                {
+                  <span>Page @content.Page</span><br/>
+                }
+                @if (!string.IsNullOrWhiteSpace(byline) && Settings.Headline.ShowByline)
+                {
+                  <span>By @byline</span><br/>
+                }
+              </div>
+            </div>
+            @if (!String.IsNullOrEmpty(body))
+            {
+              <div style="font-size:1em; line-height: 110%; margin-bottom: 10px;">@body</div>
+            }
+            @if (!string.IsNullOrEmpty(content.ImageContent) && section.Value.Settings.ShowImage)
+            {
+              var src = $"data:{content.ContentType};base64," + content.ImageContent;
+              <div><img src="@src" alt="@content.FileReferences.FirstOrDefault()?.FileName" /></div>
+            }
+            @if (Settings.Content.ShowLinkToStory && !isPrivate)
+            {
+              @* LINK TO STORY ON WEBSITE *@
+              <div>
+                <span style="font-size: .85em; text-transform:uppercase; vertical-align: middle; background-color:#FFF; border:#ccc 1px solid; margin:20 auto; padding:6px 10px;">
+                  <a href="@($"{ViewContentUrl}{content.Id}")" target="_blank" style="color: #6750a4; text-decoration: none;">View Article <img height="14" valign="absmiddle" src="@($"{SubscriberAppUrl}assets/reports/follow_link.png")"></a>
+                </span>
+              </div>
+            }
+	   else {
```

Comment:
Should this also require the `Settings.Content.ShowLinkToStory` configuration option?

Reply:
Thank you!